app/code/Magento/SalesRule/Model/ResourceModel/Rule/DeleteCustomerUsage.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\SalesRule\Model\ResourceModel\Rule;
+
+class DeleteCustomerUsage
```

Comment:
Please, cover new logic at least by unit test

Reply:
@sidolov , I am working on it